### ISSUE TITLE
AMD Drivers 22.5.2 brokes OHM reading Radeon RX 580 sensors.

### ISSUE THREAD

**Issue by UnclWish** [openhardwaremonitor/openhardwaremonitor#1515]
After update AMD Radeon Drivers to 22.5.2 (Windows 11 Pro) OHM 0.9.6 stops to show clocks of GPU and vmem, voltages, GPU loading etc.
Now it shows only temperature and power sensors.

**Comment by isaaclepes** [comment]
Same issue for me after updating to 22.5.2 on Windows 10 Pro with RX 480.  OHM just shows the name of the GPU, but no sensors.  I can still see the temp in Task Manager.

![image](https://user-images.githubusercontent.com/54421422/172013171-d56dd70b-9656-40ad-a228-97133cb7cdf1.png)

**Comment by isaaclepes** [comment]
I submitted my comment a bit too soon.. just realized I was on an older version of OHM.  Version 0.9.6 does show the sensors correctly for AMD RX 480.

![image](https://user-images.githubusercontent.com/54421422/172013361-09c3598e-e4bc-4131-a740-e8ac0f1715fd.png)

**Comment by UnclWish** [comment]
> I submitted my comment a bit too soon.. just realized I was on an older version of OHM. Version 0.9.6 does show the sensors correctly for AMD RX 480.
> 

No, not correctly! Only Temperature and power. But no frequencies, load, fan etc...

**Comment by Jlpvasconcelos** [comment]
Can someone fix this problem, or open hardware monitor is dead ???????

**Comment by joaquimorg** [comment]
Hi, I made a fix to show the missing information on RX580 and 22.5.2 drivers

![ohm_amd_rx580](https://user-images.githubusercontent.com/1682318/174168347-7bb03856-aaea-4d11-b9f9-5dce4b9023e3.jpg)

Visit my fork https://github.com/joaquimorg/openhardwaremonitor

**Comment by UnclWish** [comment]
Ohhh, it's good! Thanks!

**Comment by UnclWish** [comment]
Sorry for offtop, but what need to add in OHM reading temperature sensors from DDR3 memory modules? Crucial Bullistix.

**Comment by AndrewGG88** [comment]
Thank you very much!
